db/entities/attempt.go

```diff
@@ -5,22 +5,44 @@ import (
 	"encoding/json"
 )
 
+type Attempt struct {
+	ID            string            `json:"id" db:"id"`
+	EventId       string            `json:"event_id" db:"event_id"`
+	EndpointId    string            `json:"endpoint_id" db:"endpoint_id"`
+	Status        AttemptStatus     `json:"status" db:"status"`
+	AttemptNumber int               `json:"attempt_number" db:"attempt_number"`
+	AttemptAt     *int64            `json:"attempt_at" db:"attempt_at"` // attempted_at?
+	ErrorCode     *AttemptErrorCode `json:"error_code" db:"error_code"`
+	Request       *AttemptRequest   `json:"request" db:"request"`
+	Response      *AttemptResponse  `json:"response" db:"response"`
```

Comment:
The HTTP consumption time can also be recorded. The network response time is also a core field.

Reply:
Sounds nice. I feel that this can be done in a separate PR to keep the scope as small as possible.